README.md

```diff
@@ -220,3 +229,33 @@ This may be better suited for you if you plan to change some of the automaticall
 ## Recap
 
 After following this complete README, we should have an ETH smart contract as well as a Starknet smart contract, both connected to act as a bridge between these two chains.
+
+## Upgrade Contracts in Testnet
+
+### Ethereum
+After deploying the `YABTransfer` contract, you can perform upgrades using the `make ethereum-upgrade` command.
+
+Please note that executing this command will rebuild `YABTransfer.sol`, deploy the new contract to the network, and utilize Foundry to upgrade the proxy contract by changing its implementation to the newly deployed contract.
+
+To execute this action, you must be the **owner** of the contract, and you also need to have set the `YAB_TRANSFER_PROXY_ADDRESS` variable in the `.env` file with the proxy's address.
```

Comment:
```suggestion
After deploying the `YABTransfer` contract, you can perform upgrades of the smart contract.

To execute this action, you must be the **owner** of the contract, and you also need to have set the `YAB_TRANSFER_PROXY_ADDRESS` variable in the `.env` file with the proxy's address.
```
The command is stated more clearly and later in this Readme. I think stating commands only once makes the Readme more easily followed step-by-step.

Also, I would move the clarification of:
"Please note that executing this command will rebuild `YABTransfer.sol`, deploy the new contract to the network, and utilize Foundry to upgrade the proxy contract by changing its implementation to the newly deployed contract."
To when the command is stated, around line 261

Reply:
I think it's right to move the sentence to the line 261 :+1: 
Also, I deleted the NOTE. 

Regarding the other point, I thought that perhaps each section should explicitly state the required variables. Someone might want to go directly to the `upgrade`, and it's important to have all the necessary information in that section.